libunwind/src/Unwind-wasm.c

```diff
@@ -10,15 +10,13 @@
 //
 //===----------------------------------------------------------------------===//
 
+// clang-format off
```

Comment:
Why is this needed? Adding another newline should avoid the reordering. Alternatively, config.h should include stdbool.h to be self-contained.